### PR TITLE
BTA-2387 Update example/tests with optional params

### PR DESCRIPTION
**Pull request by archmagos**
Update example and tests to reflect the addition of optional parameters
to GET endpoints in the Java SDK.

**Resolves:** https://btcafrica.atlassian.net/browse/BTA-2387

### CODE REVIEW

File: src/test/java/com/transferzero/sdk/api/TransactionsApiTest.java [hunk 1]
```diff
@@ -82,7 +83,13 @@ public void getTransactionsTest() throws ApiException {
         Integer page = null;
         Integer per = null;
         String externalId = null;
-        TransactionListResponse response = api.getTransactions(page, per, externalId);
+        String senderId = null;
+        TransactionListResponse response = api.getTransactions()
+                .page(page)
```
Comment (sztupy) [inline src/test/java/com/transferzero/sdk/api/TransactionsApiTest.java]: if they're null no need to add them. They're optional for a reason.

File: src/test/java/com/transferzero/sdk/api/WebhooksApiTest.java [hunk 2]
```diff
@@ -95,7 +97,10 @@ public void getWebhookEventsTest() throws ApiException {
     public void getWebhooksTest() throws ApiException {
         Integer page = null;
         Integer per = null;
-        WebhookDefinitionListResponse response = api.getWebhooks(page, per);
+        WebhookDefinitionListResponse response = api.getWebhooks()
+                .page(page)
+                .per(per)
```
Comment (sztupy) [inline src/test/java/com/transferzero/sdk/api/WebhooksApiTest.java]: similarly, either make the examples useful by specifying details here, or just remove the calls altogether

**Comment by sztupy** [comment]
I can't seem to see the changes in the `src/main` directory, only the tests have gotten the new features

**Comment by archmagos** [comment]
@sztupy are the `src/main` files not automatically generated and committed by the OpenApi generator when the specification is updated? I assumed that was the case but will commit those changes if not (likewise the documentation).

**Comment by sztupy** [comment]
Well, the build is failing anyway so there's something off

File: example/src/main/java/com/transferzero/test/Application.java [hunk 3]
```diff
@@ -411,7 +413,9 @@ public static void getSenderByExternalId(ApiClient apiClient) throws Exception {
         SendersApi sendersApi = new SendersApi(apiClient);
         String externalId = "SENDER-2b59defy";
         try {
-            SenderListResponse senderListResponse = sendersApi.getSenders(null, null, null, null, externalId);
+            SenderListResponse senderListResponse = sendersApi.getSenders()
```
Comment (sztupy) [inline example/src/main/java/com/transferzero/test/Application.java]: soo much nicer 👍

**Pull request merged**